src/filters/medianFilter.ts

```diff
@@ -0,0 +1,74 @@
+import { xMedian } from 'ml-spectra-processing';
+
+import { Image } from '../Image';
+import checkProcessable from '../utils/checkProcessable';
+import { getBorderInterpolation, BorderType } from '../utils/interpolateBorder';
+
+export interface MedianFilterOptions {
+  /**
+   * Type of border algorithm to interpolate from.
+   *
+   * @default 'reflect101'
+   */
+  borderType: BorderType;
+  /**
+   * Value of border.
+   */
+  borderValue: number;
```

Comment:
In other methods of image-js having the `borderType` option, is this option also a required field?